taxcalc/consumption.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_MPC_e17500": {
+        "start_year": 2013,
+        "long_name": "Marginal propensity to consume medical expenses",
+        "description": "Defined as dollar change in medical-expense consumption divided by dollar change in income.  Must be in [0,1] range.",
```

Comment:
@martinholmer, why is it important to constrain the MPCs to [0,1]? My intuition is that some MPCs could be negative (are these inferior goods?) or in some cases they could be greater than 1 (perhaps an addictive drug that causes you to draw down savings once you get started)? The MPC's for these items probably are in [0,1], but why constrain it?